vm-to-vm-bandwidth-meter/azuredeploy.json

```diff
@@ -0,0 +1,611 @@
+        "256k",
+        "512k",
+        "1m"
+      ],
+      "defaultValue": "8k",
+      "metadata": {
+        "description": "Size of the request to send to the target VM."
+      }
+    },
+    "testRequestCount": {
+      "type": "int",
+      "defaultValue": 1000,
+      "metadata": {
+        "description": "Number of the requests to send to the target VM."
+      }
+    },
+      "adminUsername": {
+        "type": "string",
+        "metadata": {
+          "description": "Username for the probe and target VMs."
+        }
+      },
+      "adminPassword": {
+        "type": "securestring",
+        "metadata": {
+          "description": "Password for the probe and target VMs."
+        }
+      },
+      "_artifactsLocation": {
+        "type": "string",
+        "metadata": {
+          "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+        },
+        "defaultValue": "https://raw.githubusercontent.com/AlekseiPolkovnikov/azure-quickstart-templates/bw/vm-to-vm-bandwidth-meter"
+      },
+      "_artifactsLocationSasToken": {
+        "type": "securestring",
+        "metadata": {
+          "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+        },
+        "defaultValue": ""
+      }
+    },
+  "variables": {
+    "windowsOSVersion": [
+      "2012-R2-Datacenter",
+      "14.04.5-LTS"
+    ],
+    "imagePublisher": [
+      "MicrosoftWindowsServer",
+      "Canonical"
+    ],
+    "imageOffer": [
+      "WindowsServer",
+      "UbuntuServer"
+    ],
+    "OSDiskName": "osdiskforwindowssimple",
+    "nicName": "myVMNic",
+    "addressPrefix": "10.0.0.0/16",
+    "subnetName": "Subnet",
+    "subnetPrefix": "10.0.0.0/24",
+    "storageAccountType": "Standard_LRS",
+    "publicIPAddressName": "publicIP",
+    "publicIPAddressType": "Static",
+    "vmStorageAccountContainerName": "vhds",
+    "virtualNetworkName": "testVNET",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "scriptFolder": "scripts",
+    "scriptFileName": "testBandwidth.ps1",
+    "vhdStorageAccountName": "[concat(uniqueString(resourceGroup().id), 'storage')]",
+    "frontEndNSGName": "[concat('webtestnsg-', uniqueString(resourceGroup().id))]",
+    "location": [
+      "[parameters('probeVmLocation')]",
+      "[parameters('targetVmLocation')]"
+    ],
+    "vmName": [
+      "probeVM",
+      "targetVM"
+    ],
+    "subnetRef": [
+      "[concat(variables('vnetID'),'1','/subnets/',variables('subnetName'))]",
+      "[concat(variables('vnetID'),'2','/subnets/',variables('subnetName'))]"
+    ]
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Storage/storageAccounts",
```

Comment:
can we update this to the new schema: https://github.com/Azure/azure-quickstart-templates/blob/master/101-storage-account-create/azuredeploy.json